guides/common/modules/proc_creating-an-installation-medium-for-debian.adoc

```diff
@@ -11,7 +11,7 @@ endif::[]
 . In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Setup* > *Installation Media*.
 . Click *Create Medium* to create an installation medium entry.
 . Enter a *Name* for the installation medium.
-. Set the *Path* to the upstream URL `{client-debian-installation-medium-path}`.
```

Comment:
I think that paths should be represented as literals. What is the reason not to?